14-draft.json

```diff
@@ -241,6 +241,10 @@
           "description": "Facebook account URL.",
           "type": "string"
         },
+        "instagram": {
+          "description": "Instagram account handle.",
+          "type": "string"
+        },
```

Comment:
Do spaces use Instagram as a way to contact them? I thought Instagram is mainly used by spaces as a way to share content, not as a channel to be contacted.

I'm asking the question because adding every popular service to this list will end up in a huge list over time. The "contact" section is mostly meant for "ways to contact this space".

@rnestler @gidsi Maybe we should add a new section called "links" that can contain free-form key-value links? For example:

```
"links": {
    "instagram": "https://www.instagram.com/yourspace",
    "that-other-cool-service": "https://www.instagram.com/yourspace"
}
```

Reply:
That's a good point actually. We do mainly use it for sharing cool projects and community engagement. It might be better off under `projects` then I guess?

Reply:
> Do spaces use Instagram as a way to contact them? I thought Instagram is mainly used by spaces as a way to share content, not as a channel to be contacted.

I agree, Instagram is not a way of bidirectional communication as far as I know.

> @rnestler @gidsi Maybe we should add a new section called "links" that can contain free-form key-value links? For example:

I'm slightly biased to not add new fields and `projects` already allows to add arbitrary links.

Reply:
Ah, I forgot about `projects`. Yeah, if the account is mainly used for showing projects, that might be a good place to put it.

In the long term, adding a field for "general links" (with name, description and URL) might be a good idea to avoid cluttering other fields that aren't meant for online profiles.

Reply:
There are definitely people using instagram as a contact option. I've seen a few YouTubers and Tattoo Artists that are even using it as the only contact option. 

That said, i think we should refactor that whole section and make it more dynamic. 
We also mix contact/communication/space access/social media here too.

My first thought is to remove everything beside phone and email and distribute the other keys to something more descriptive (you should also be able to add more phone numbers and email addresses and be able to add a contact purpose like space access, media contact, general etc.).